.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
Some comment in here about what this file is trying to do and how it does it would be good.